figures/client-reconnect-sequence.mmd

```diff
@@ -0,0 +1,9 @@
+sequenceDiagram
+    Actor Client
+    Client->>+OTT: Joins Through Balancer
+    OTT-->>+Client: Join Event
+    OTT->>-OTT: Stores Reference to Client
+    Client-->>+OTT: Disconnects
+    OTT->>OTT: Check Client WebSocket Disconnect Code
+    Client->>OTT: Reconnects in Specified Timeframe
+    OTT-->>-Client: No Join Event on Reconnects
```

Comment:
The "no join" should probably be a note instead to show that the join message is not being sent.